pkg/occlient/occlient.go

```diff
@@ -2076,6 +2076,63 @@ type Service struct {
 	PlanList []string
 }
 
+// GetServiceClass retrieves the ClusterServiceClass identified by the specified external name
+func (c *Client) GetServiceClass(externalName string) (class *scv1beta1.ClusterServiceClass, err error) {
+	classes, err := c.serviceCatalogClient.ClusterServiceClasses().List(metav1.ListOptions{
+		FieldSelector: "spec.externalName==" + externalName,
+	})
+
+	if len(classes.Items) != 1 {
+		return nil, nil
+	}
+
+	return &classes.Items[0], err
+}
+
+// GetServiceClassesByCategory retrieves a map associating category name to ClusterServiceClasses matching the category
+func (c *Client) GetServiceClassesByCategory() (categories map[string][]scv1beta1.ClusterServiceClass, err error) {
+	categories = make(map[string][]scv1beta1.ClusterServiceClass)
+	classes, err := c.GetClusterServiceClasses()
+	if err != nil {
+		return nil, errors.Wrap(err, "unable to get cluster service classes")
+	}
+
+	// todo: Should we replicate the classification performed in
+	// https://github.com/openshift/console/blob/master/frontend/public/components/utils/categorize-catalog-items.ts?
```

Comment:
link does not work

Reply:
Indeed, looks like the file got removed by https://github.com/openshift/console/commit/43b52896201e160078995536750377b64e114d9f#diff-852178fde0de1b406e979937572f24cf, need to figure out how it was replaced.

Reply:
Fixed.